CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Smart Collections - Users can enable search on Smart Collections
```

Comment:
`- Smart Collections - Add a new isSearchable property to display the search bar for Smart Collections.`